site/docs/flink.md

```diff
@@ -18,24 +18,24 @@
 # Flink
 
 Apache Iceberg supports both [Apache Flink](https://flink.apache.org/)'s DataStream API and Table API to write records into an Iceberg table. Currently,
-we only integrate Iceberg with Apache Flink 1.11.x.
-
-| Feature support                                                        |  Flink 1.11.0      |  Notes                                                 |
-|------------------------------------------------------------------------|--------------------|--------------------------------------------------------|
-| [SQL create catalog](#creating-catalogs-and-using-catalogs)            | ✔️                 |                                                        |
-| [SQL create database](#create-database)                                | ✔️                 |                                                        |
-| [SQL create table](#create-table)                                      | ✔️                 |                                                        |
-| [SQL create table like](#create-table-like)                            | ✔️                 |                                                        |
-| [SQL alter table](#alter-table)                                        | ✔️                 | Only support altering table properties, Columns/PartitionKey changes are not supported now|
-| [SQL drop_table](#drop-table)                                          | ✔️                 |                                                        |
-| [SQL select](#querying-with-sql)                                       | ✔️                 | Support both streaming and batch mode                  |
-| [SQL insert into](#insert-into)                                        | ✔️ ️               | Support both streaming and batch mode                  |
-| [SQL insert overwrite](#insert-overwrite)                              | ✔️ ️               |                                                        |
-| [DataStream read](#reading-with-datastream)                            | ✔️ ️               |                                                        |
-| [DataStream append](#appending-data)                                   | ✔️ ️               |                                                        |
-| [DataStream overwrite](#overwrite-data)                                | ✔️ ️               |                                                        |
-| [Metadata tables](#inspecting-tables)                                  |    ️               | Support Java API but does not support Flink SQL        |
-| [Rewrite files action](#rewrite-files-action)                          | ✔️ ️               |                                                        |
+we only integrate Iceberg with Apache Flink 1.11.x\1.12.x\1.13.x\1.14.x
```

Comment:
Is 1.11 supported? I thought that was dropped and only 1.12, 1.13, and 1.14 are supported.

Reply:
@hililiwei can you comment on this?

Reply:
At least in 0.11, we support flink 1.11. During technical selection, our team verified the compatibility. However, the latest version of the code does remove the 1.11 support. I'm not quite sure what the reason for the removal is. If we need to keep version 1.11, I think we need to create a module for 1.11 and verify compatibility.Do we need to ensure the latest version is compatible with 1.11? If so, maybe we need to create a 1.11 module and verify compatibility.

Reply:
refer https://github.com/apache/iceberg/pull/1956